README.md

```diff
@@ -7,12 +7,65 @@ MyHebrewDates is a web app for sharing Hebrew dates like birthdays, anniversarie
 
 License: MIT
 
+## Contributors
+<div align="center">
+<a href="https://github.com/abe-101/myHebrewDates/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=abe-101/myHebrewDates" />
+</a>
+</div>
+
 ## Settings
 
 Moved to [settings](http://cookiecutter-django.readthedocs.io/en/latest/settings.html).
 
 ## Basic Commands
 
+## Installation
+
+1. Clone the repository:
+  ```shell
+  git clone https://github.com/abe-101/myHebrewDates.git
+  cd myHebrewDates
+  ```
+
+2. Create and activate a virtual environment:
+  ```shell
+  python3 -m venv venv
+  source venv/bin/activate
+  ```
+
+3. Install the local dependencies:
+  ```shell
+  pip install -r requirements/local.txt
+  ```
+
+4. Pre-Commit Install:
+  ```shell
+  pre-commit install
+  ```
+
+5. Create Database:
+  ```shell
+  createdb --username=<USERNAME> my_hebrew_dates
+  ```
+
+6. Create Env Files and add these:
+  ```shell
+  DATABASE_URL=postgres://<USERNAME>:<PASSWORD>@127.0.0.1:5432/my_hebrew_dates
+  CELERY_BROKER_URL=redis://localhost:6379/0
+  ```
+
+7. Set up the database:
+  ```shell
+  python manage.py makemigrations
+  python manage.py migrate
+  ```
+
+8. Start the Server
+  ```shell
+  python manage.py runserver
+  ```
+
 ### Setting Up Your Users
 
 - To create a **normal user account**, just go to Sign Up and fill out the form. Once you submit it, you'll see a "Verify Your E-mail Address" page. Go to your console to see a simulated email verification message. Copy the link into your browser. Now the user's email should be verified and ready to go.
```

Comment:
i think we can remove 
### Type checks 
and
### Test coverage

Reply:
sry could u specify the line number or point number

Reply:
we can remove lines 79 though 127 (new line numbers)

Reply:
@seabeePraveen Can you make the final fix's and I'll merge?

Reply:
yeah yeah i will make the final fix's
and sry to ask again like i found the setup was fine so could u please specify why these changes are required for those lines u have mentioned also please mention from which line to which line i have to change